### PR TITLE
Do 9037 ecs logstash

### PR DESCRIPTION
**Pull request by alexandru-rusanda**

### CODE REVIEW

File: templates/logstash.json.tpl [hunk 1]
```diff
@@ -20,27 +20,28 @@
...
     "entryPoint": [],
+    "requiresCompatibilities": ["FARGATE"],
     "environment": [
         {
           "name": "ENV_NAME",
           "value": "${ENV_NAME}"
         },
         {
-          "name": "LOGSTASH_HOST",
+          "name": "ES_URL",
```
Comment (saritdevops) [inline templates/logstash.json.tpl]: @alexandru-rusanda should be removed -- no ES_URL anymore.

File: templates/logstash.json.tpl [hunk 2]
```diff
@@ -20,27 +20,28 @@
         "options": {
             "awslogs-group": "/ecs/td-${ENV_NAME}-logstash",
             "awslogs-region": "us-east-1",
+            "awslogs-create-group": "true",
             "awslogs-stream-prefix": "ecs"
           }
         },
-    "memoryReservation": 128,
```
Comment (saritdevops) [inline templates/logstash.json.tpl:26]: i thought memory is 20G?

**Review by saritdevops — APPROVED** [review]
Besides my comments the module changes look good. please fix them and merge!

File: templates/logstash.json.tpl (hunk 2 above)
Comment (alexandru-rusanda) [reply templates/logstash.json.tpl:26]: From the doc
 Soft limit (in MiB) of memory to reserve for the container. When system memory is under contention, Docker attempts to keep the container memory to this soft limit.
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ecs_container_definition

Memory is defined in resource "aws_ecs_task_definition" in main.tf file.

**Pull request merged**